packages/lib-classifier/src/components/Classifier/components/SubTaskPopup/SubTaskPopup.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import counterpart from 'counterpart'
+import { Box, Paragraph } from 'grommet'
+import { Rnd } from 'react-rnd'
+import { inject, observer } from 'mobx-react'
+import {} from 'prop-types'  // TODO
+
+import SingleChoiceTask from '@plugins/tasks/SingleChoiceTask'
+import taskRegistry from '@plugins/tasks'
+
+import styled from 'styled-components'  // TODO: check what's the best way to style this component
+import zooTheme from '@zooniverse/grommet-theme'
+
+
+
+// EXPERIMENTAL
+// ----------------
+const StyledBox = styled(Box)`
+  border: 2px solid ${zooTheme.global.colors.brand}
+`
+// ----------------
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function storeMapper (stores) {
+  const { activeStepTasks } = stores.classifierStore.workflowSteps
+  const [activeDrawingTask] = activeStepTasks.filter(task => task.type === 'drawing')
+  const activeMark = (activeDrawingTask && activeDrawingTask.activeMark)
+    ? activeDrawingTask.activeMark
+    : undefined
+  const { addAnnotation } = stores.classifierStore.classifications
```

Comment:
```suggestion
  const { addAnnotation } = activeMark
```
You'll want to add these annotations to the active mark, not to the classification.

Reply:
Whoops! Thanks for spotting this, I forgot to clean this up from the previous PR since `props.addAnnotation` is no longer used; I pull `props.activeMark.addAnnotation` straight during the render()